src/contexts/W3iContext/hooks/pushHooks.ts

```diff
@@ -42,7 +42,10 @@ export const usePushState = (w3iProxy: Web3InboxProxy, proxyReady: boolean, dapp
   }, [pushClient, userPubkey])
 
   useEffect(() => {
-    refreshPushState()
+    // Account for sync init
+    const timeoutId = setTimeout(() => refreshPushState(), 100)
```

Comment:
What event are we missing to handle this properly?

Reply:
We need something like `sync_stores_initialized` that accounts for the init of `subscriptions`. This is due to the fact that way stores are initialized is we first init a `Store` instance for `subscriptions`. Then, after sync keys are updated etc we initialize a `SyncStore` instance. There is no event for that